api/lib/application/organizations/organization-controller.js

```diff
@@ -103,9 +103,11 @@ module.exports = {
   importSchoolingRegistrationsFromSIECLE(request) {
     const organizationId = parseInt(request.params.id);
     const buffer = request.payload;
-
+    console.time('endpoint');
```

Comment:
Est-ce qu'on veut garder ces mesures en prod ?
Si oui, est-ce qu'on ne devrait pas passer par le `logger` pour avoir des logs correctement formatés dans Datadog ?
De plus, `endpoint` semble trop générique, il vaudrait mieux spécifier la fonction sous test, par exemple :
```suggestion
    console.time('endpoint -  importSchoolingRegistrationsFromSIECLE');
```

Reply:
C'était pas prévue de les laisser en prod 😉